docs/documentation/server_admin/topics/threat/fapi-compliance.adoc

```diff
@@ -2,6 +2,6 @@
 === FAPI compliance
 
 To make sure that {project_name} server will validate your client to be more secure and FAPI compliant, you can configure client policies
-for the FAPI support. Details are described in the FAPI section of link:{adapterguide_link}#_fapi-support[{adapterguide_name}]. Among other things, this ensures some security best practices
+for the FAPI support. *FAPI* details are described in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section. Among other things, this ensures some security best practices
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).